docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -189,29 +190,49 @@ Advanced flags can be set by using the configuration file in the `--config` flag
 
 ### configure
 
-Use the `yugabyted configure` command to configure the data placement constraints for the YugabyteDB cluster.
+Use the `yugabyted configure` command to do the following:
+
+* Configure the data placement policy of the cluster.
+* Enable or disable encryption at rest.
+
+#### Commands
+
+The following subcommands are available for `yugabyted configure` command:
+
+- [data_placement](#data-placement)
+- [encrypt_at_rest](#encrypt-at-rest)
 
 #### Syntax
 
-```text
-Usage: yugabyted configure [flags]
+```sh
+Usage: yugabyted configure [command] [flags]
+```
+
+#### data_placement
+
+Use the `yugabyted configure data_placement` subcommand to set or modify placement policy of the nodes of the deployed cluster.
+
+#### Syntax
+
+```sh
+Usage: yugabyted configure data_placement [flags]
 ```
 
-For example, to create a multi zone YugabyteDB cluster:
+For example, you would use the following command to create a multi-zone YugabyteDB cluster:
 
 ```sh
-yugabyted configure --fault_tolerance=zone
+yugabyted configure data_placement --fault_tolerance=zone
 ```
 
 #### Flags
 
 -h | --help
 : Print the command line help and exit.
 
---fault_tolerance *fault_tolerance*
+--fault_tolerance *fault-tolerance*
 : Specify the fault tolerance for the cluster. This flag can accept one of these values - zone, region, or cloud. For example, when the flag is set to zone (`--fault_tolerance=zone`), yugabyted applies zone fault tolerance to the cluster, placing the nodes in 3 different zones, if available.
```

Comment:
Specify the fault tolerance for the cluster. This flag can accept one of the following values: zone, region, cloud. For example, when the flag is set to zone (`--fault_tolerance=zone`), yugabyted applies zone fault tolerance to the cluster, placing the nodes in three different zones, if available.